src/pizza/Order.java

```diff
@@ -0,0 +1,79 @@
+package pizza;
+
+import java.util.Arrays;
+
+public class Order {
+
+    private final int MAX_ORDER_ITEM_COUNT = 10;
+    private int orderNumber;
+    private Customer customer;
+    private OrderItem[] orderItems;
+    private int orderItemCount = 0;
+
+
+    public Order() {
+        orderItems = new OrderItem[MAX_ORDER_ITEM_COUNT];
+    }
+
+    public Order (int orderNumber, Customer customer, OrderItem[] orderItems) {
+        for (int i = 0; i < orderItems.length; i++) {
+            OrderItem orderItem = orderItems[i];
+            if (orderItem != null) {
+                String pizzaName = checkOfName(orderItem.getPizza().getName());
+                orderItem.getPizza().setName(pizzaName);
+            }
+        }
+        this.orderNumber = orderNumber;
+        this.customer = customer;
+        this.orderItems = orderItems;
+    }
+
+    public Order(Customer customer, OrderItem[] orderItems) {
+    }
+
+    public static Order addOrder(int orderNumber, Customer customer, OrderItem ... orderItems){
+        return new Order(orderNumber, customer,orderItems);
+    }
+
+    private String checkOfName(String pizzaName) {
+        String result;
+
+        if(pizzaName.length() > 4 && pizzaName.length() < 20){
+            result = pizzaName;
+        } else {result = "customer_name_10000";
```

Comment:
You don't need to put fixed value. You have to get customerNumber in the method's parameter.